core/ui/src/main/java/com/guidal/core/ui/theme/Icon.kt

```diff
@@ -33,7 +33,15 @@ object GuidalIcons {
         val ChevronForward: ImageVector,
         val Search: ImageVector,
         val ArrowBack: ImageVector,
-        val ArrowForward: ImageVector
+        val ArrowForward: ImageVector,
+        val Museum: ImageVector,
+        val Beach: ImageVector,
+        val Commute: ImageVector,
+        val Hiking: ImageVector,
+        val NightLife: ImageVector,
+        val Restaurant: ImageVector,
+        val Shop: ImageVector,
+        val CircledStar: ImageVector
```

Comment:
Would be better to define as a seperate Category object, similar to how genders and countries have their object of icons.

Reply:
I did this before the updates in the main branch. Will redefine.